generators/client/templates/angular/tslint.json.ejs

```diff
@@ -21,11 +21,6 @@
         "node_modules/codelyzer"
     ],
     "rules": {
-        "member-ordering": [
```

Comment:
shouldn't the file be removed?

Reply:
no, we still need this to include `codelyzer` (angular specific) `tslint` rules.